docs/source/users/installation.rst

```diff
@@ -0,0 +1,99 @@
+Install and configure wagtail-localize-panel
+============================================
+
+
+Installation
+------------
+
+`wagtail-localize-panel` is available on pypi, you can install with your favorite
```

Comment:
```suggestion
``wagtail-localize-panel`` is available on pypi, you can install with your favorite
```

Instead no (and idem for the rest of the doc)?

Reply:
```suggestion
``wagtail-localize-panel`` is available on pypi, you can install it with your favorite
```